server/sonar-server/src/main/resources/org/sonar/server/organization/ws/example-create.json

```diff
@@ -0,0 +1,10 @@
+{
+  "organization": {
+    "uuid": "AU-Tpxb--iU5OvuD2FLy",
+    "key": "foo-company",
+    "name": "Foo Company",
+    "description": "The Foo company produces quality software for Bar.",
+    "url": "https://www.foo.com",
+    "avatar": "https://www.foo.com/foo.png"
```

Comment:
just to be consistent with database, should it be renamed `avatarUrl` ? Or database column to be renamed `avatar` ?